lib/matplotlib/tests/test_axes.py

```diff
@@ -351,7 +349,7 @@ def test_shaped_data():
     plt.plot(y2)
 
     plt.subplot(413)
-    assert_raises(ValueError, plt.plot, (y1, y2))
+    pytest.raises(ValueError, plt.plot(y1, y2))
```

Comment:
I don't think your changes to the raise tests make sense. This line currently means "call `plt.plot` and pass the result to `pytest.raises`", then `pytest.raises` will attempt to call _that_, which generally doesn't make sense. `pytest.raises` accepts `*args, **kwargs`, so I think this one should be `pytest.raises(ValueError, plt.plot, y1, y2)`.